README.md

```diff
@@ -308,7 +308,7 @@ New extensions are becoming available on a regular basis!
 
 ## Configuration
 
-Config file is passed to the `phpstan` executable with `-c` option:
+A config file can passed to the `phpstan` executable using the `-c` option:
```

Comment:
`can be` - `be` is missed

Reply:
still the issue

Reply:
Feel free to send a PR :)

Reply:
Okay

https://github.com/phpstan/phpstan/pull/3103